homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
```

Comment:
Ideally we split out the `extra_state_attributes` in separate entities which can be more contextualized